spring-integration-jdbc/src/test/java/org/springframework/integration/jdbc/store/channel/DataSource-postgres-context.xml

```diff
@@ -10,14 +10,18 @@
 
 	<bean id="dataSource" class="org.apache.commons.dbcp2.BasicDataSource"
 		destroy-method="close">
-		<property name="driverClassName" value="org.postgresql.Driver" />
+		<property name="driverClassName" value="org.testcontainers.jdbc.ContainerDatabaseDriver" />
 		<property name="initialSize" value="10"/>
-		<property name="url"
-			value="jdbc:postgresql:integration" />
+		<property name="url" value="jdbc:tc:postgresql:13://hostname/testdb" />
```

Comment:
I'm not sure what is this and how that suppose to work in our test.
Please, consider to revert changes to these XML configs.
And let's see if you can implement the test based on our `PostgresContainerTest` and already with only Java configuration!

Reply:
This is a jdbc url that is resolve by testcontainers. It will automagically start a postgres. I didn't had a local postgres running and I wanted to keep the test similar to the others and don't overdo the entire setup for those tests.

Reply:
That sounds like a nice feature, but I want to have a container instance re-used when we run the whole suite.
Well, it can be done as a subsequent change, though. 
Never mind for now: thank you for education!

Reply:
Done.